docs/operator/k8s_guide.md

```diff
@@ -0,0 +1,725 @@
+  template:
+    data: null
+    metadata:
+      creationTimestamp: null
+      name: rucio-servers-hostcert
+      namespace: rucio
+```
+## Rucio Servers
+[Reference Helm chart](https://gitlab.cern.ch/rucio-it/flux-compass/-/blob/master/sync/rucio-servers.yaml?ref_type=heads).
+
+>The Rucio servers are the backbone of the Rucio system. They handle all core functionalities including data management, rule-based data replication, data placement, and monitoring. The servers ensure the integrity and availability of data across various storage systems.
+
+In order to setup this service, four steps are needed:
+1. Produce the Helm chart
+2. Setup the LB
+3. Produce the certificates related to the landb-alias used
+4. Add the certificates as secrets and mount them on the k8s pods.
+
+### Produce the Helm chart
+Please look at the [currently used one](https://gitlab.cern.ch/rucio-it/flux-compass/-/blob/master/sync/rucio-servers.yaml?ref_type=heads).
+A few remarks:
+
+- The db secret is being mounted on `config.database.default` in order to allow the user to correctly access it.
+- `hostcert.pem`, `hostkey.pem` and `ca.pem` are automatically detected, so they don't need to be mounted. Nevertheless, the corresponding secrets must be created (see next sections).
+- The GridCA cert is needed to verify all the other certs. Please look at this [git commit](https://gitlab.cern.ch/rucio-it/flux-compass/-/commit/688da7285833dafd1bbe25469f35c6059d7af24f) used to set the `GridCA` file in `/etc/grid-security/certificate`.
+	- Together with this, is also necessary to setup the correspoding variable `RUCIO_CA_PATH`
+- The `RUCIO_SSL_PROTOCOL` variable must be explicitly set.
+
+Please notice that in order to have LBs produced, ***the Helm chart must be applied***. 
+This will come with several errors related to certificates, that will be fixed in the next steps.
+### Setup the LB
+The minimal configuration for LBaaS is the following:
+```yaml
+    service:
+      type: LoadBalancer
+      port: 443
+      targetPort: 443
+      protocol: TCP
+      name: https
+```
+Once the Helm chart is applied, it will trigger the `openstack-cloud-controller-manager` pod, that will automatically instantiate the requested LBs. 
+
+Please notice that in order to have LBs, one must request a quota change to the openstack project:
+![[/img/get-lbs.png]]
+
+To check the status of the LBs, the CLI command can be used: 
+```sh
+openstack loadbalancer list
+```
+
+Then, a few attributes must be changed in order to obtain the necessary DNS:
+```sh
+openstack loadbalancer set --tag landb-set="RUCIO-IT" <lb-ID>
+openstack loadbalancer set --tag landb-alias="compass-rucio.cern.ch" <lb-ID>
+openstack loadbalancer set --description="compass-rucio.cern.ch" <lb-ID>
+openstack loadbalancer set --tag main-user="rucio-it-admins" <lb-ID>
+```
+
+Now, the LB must be added to the LanDB set from the dashboard, under IP Addresses.
+To do that, first retrieve the `vip_address` of the LB:
```

Comment:
vip_address should be defined if possible

Reply:
Done.